File: Programming-languages-and-laboratory/Python/Exercises/41_compito_java_20221110/Servizio.py
class Servizio:
    def __init__(self, codice_dipendente, ore_servizio):
        self._codice_dipendente = codice_dipendente
        self._ore_servizio = ore_servizio

    def __str__(self):
        return str(self._codice_dipendente) + ", " + str(self._ore_servizio)

File: Programming-languages-and-laboratory/Java/Exercises/41_compito_java_2022_11_10/Servizio.java
public class Servizio {
	protected Dipendente dip;
	protected float oreServizio;
	
	public Servizio(Dipendente dip, float oreServizio) {
		this.dip = dip;
		this.oreServizio = oreServizio;
	}
	
	public String toString() {
		return "(" + dip.codice + ", " + oreServizio + ")";
	}
}